cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/manager/DynamoDbTemplateInfrastructureManager.java

```diff
@@ -33,24 +27,14 @@
  *
  * The responsibility of implementing classes is to create tables
  */
-public class DynamoDbTemplateInfrastructureManager {
+public class DynamoDbTemplateInfrastructureManager extends AbstractPersistenceManager {
```

Comment:
Would be better to leave the DynamoDbTemplateInfrastructureManager as it was and make the DynamoDbTemplate and DynamoDocumentStoreTemplate implements AbstractDynamoDbTemplate which has the common elements. Then we should make the DynamoDbTemplateInfrastructureManager make use of AbstractDynamoDbTemplate instead of DynamoDbTemplate.